packages/daylight/system/pid.go

```diff
@@ -0,0 +1,50 @@
+package system
```

Comment:
I personally hate this stuff, thats not unix-way, pid file should contain only pid)